app/routes/datasets/asset.py

```diff
@@ -94,7 +94,12 @@ async def add_new_asset(
         await verify_asset_dependencies(dataset, version, input_data["asset_type"])
 
     try:
-        asset_uri = get_asset_uri(dataset, version, input_data["asset_type"])
+        asset_uri = get_asset_uri(
+            dataset,
+            version,
+            input_data["asset_type"],
+            input_data.get("creation_options"),
```

Comment:
Let's stay consistent on how we request dict items.
`.get` makes sense for me, if I am not certain if it exists and if I want to define a default value. Otherwise the `[]` works just fine. Other thoughts?

Reply:
Hmm, not sure why I used `get` here. I agree, will fix. I must at some point have thought that creation_options was optional, but it's not.